lib/internal/test_runner/utils.js

```diff
@@ -412,6 +413,19 @@ function getCoverageReport(pad, summary, symbol, color, table) {
   return report;
 }
 
+// In certain places, # and \ need to be escaped as \# and \\.
+function tapEscape(input) {
```

Comment:
since this is used in spec and no longer related only to tap, can you rename it to something more generic?

Reply:
maybe `parseEscape` or `parseEscapeSequence`? 🤔